sources/config/Context.swift

```diff
@@ -53,7 +53,7 @@ extension VVPSDK {
                     model: hardwareVersion,
                     os: UIDevice.current.systemVersion),
                 sdk: SDK.init(
-                    version: "2.31",
+                    version: "2.32",
```

Comment:
I think it's better to move all our versions string into a separate file. @RomanTysiachnik @AndriiMoskvin What do you think

Reply:
Sounds good. We already have `VVPSDK+Version` file.